src/h5web/toolbar/controls/DomainSlider/DataHistogram.tsx

```diff
@@ -0,0 +1,49 @@
+import { AxisBottom, AxisLeft } from '@visx/axis';
+import { scaleLinear } from '@visx/scale';
+import { useDomain } from '../../../vis-packs/core/hooks';
+import type { Size } from '../../../vis-packs/core/models';
+import styles from './DataHistogram.module.css';
+
+interface Props {
+  bins: number[];
+  data: number[];
+  size: Size | undefined;
```

Comment:
You could pass `width` and `height` directly? You'd then have `type Props = HistogramData & Size`.